app/modules/annotations/models.py

```diff
@@ -480,3 +483,27 @@ def get_elasticsearch_schema(cls):
         from app.modules.annotations.schemas import AnnotationElasticsearchSchema
 
         return AnnotationElasticsearchSchema
+
+    def send_to_identification(self, matching_set_query=None):
```

Comment:
This is what I said should be in the sightings module. Then it could be used for ia_pipeline, annot ID and encounter ID. 
Is there any reason you have not done so that I'm missing?